build/openwisp_radius/Dockerfile

```diff
@@ -1,15 +1,22 @@
 FROM python:3.7-alpine AS BASE
 
-WORKDIR /opt/openwisp/
-RUN apk add  --update --no-cache build-base postgresql-dev libffi-dev openssl-dev python3-dev zlib-dev jpeg-dev git gettext linux-headers
-RUN pip install https://github.com/openwisp/django-freeradius/tarball/master && \
+RUN apk add  --update --no-cache \
+    # TODO: purge the build-deps after python installation.
+    libffi-dev build-base postgresql-dev openssl-dev python3-dev zlib-dev jpeg-dev \
+    git gettext linux-headers && \
```

Comment:
it seems many of these packages are defined in each image, I think we need a base image from which all images inherit from to avoid this repetition. Building may also be faster if we don't change the base image often.

@atb00ker what do you think? In some projects I work on, we almost always have a base image.

Reply:
I believe postgis dependencies for alpine are only available in edge.

Reply:
I am working on the base image right now. :-)